### ISSUE TITLE
New feature: Vimeo support

### ISSUE THREAD

**Issue by jenda77** [contao/core#8219]
It would be very nice to have the possibility to add vimeo videos like we can do it with youtube videos now.

**Comment by aschempp** [comment]
:+1: the necessary CE is pretty simple if we agree on the iframe (as we should for youtube, but thats another topic…).

```php
<iframe src="//player.vimeo.com/video/<?= $this->vimeo ?>" width="<?= $this->width ?>" height="<?= $this->height ?>" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
```

see https://developer.vimeo.com/player/embedding

**Comment by jenda77** [comment]
:+1: That would be great.

**Comment by leofeyer** [comment]
Implemented in contao/core-bundle@ccde5ae3daaa6fcd57da9b88e4d5ec49fe89da18.

**Comment by jenda77** [comment]
Great. Thank you very much.
